api/graphql/base_serializers.py

```diff
@@ -7,9 +10,30 @@ def _check_id_list(self, id_list, field_name):
             try:
                 int(identifier)
             except ValueError:
-                raise serializers.ValidationError(
-                    f"Wrong type of id: {identifier} for {field_name}"
-                )
+                raise GraphQLError(f"Wrong type of id: {identifier} for {field_name}")
+
+    def to_internal_value(self, data):
+        try:
+            int_val = super().to_internal_value(data)
+        except serializers.ValidationError as e:
+            raise self.validation_error_to_graphql_error(e)
+
+        return int_val
+
+    def validation_error_to_graphql_error(self, e: serializers.ValidationError):
+        field = "nonFieldError"
+        message = ""
+        if isinstance(e.detail, dict) and len(e.detail.items()) > 0:
+            for f, detail in e.detail.items():
+                field = to_camel_case(f)
```

Comment:
No, actually first is to be intended. Raising multiple errors isn't possible so we would need to get a other approach then.
So I'll fix this to return the first encountered error.

Reply:
Intended behaviour is to have the errors listed. I merged all the validation errors to graphqlerror.
Obvioysly it would be best to get list of errors but I did not found the way.